### PR TITLE
としくんのかっこよさについて語るスレ

### PR DESCRIPTION
**Pull request by tamamu**
Respect for ystk!!!!!!
He is cool guy!!!!!!

### CODE REVIEW

**Review by kokoax — APPROVED** [review]
ystk is god

**Comment by tamamu** [comment]
#1 is unchi :hankey:

**Comment by kokoax** [comment]
the unchi is Fock

File: toshi.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# CHAGU CHAGU UMAKKO
```
Comment (technicakidz) [inline toshi.md:1]: なーにがumakkoじゃ

**Comment by technicakidz** [comment]
LGTM!

**Comment by technicakidz** [comment]
LGTM

**Pull request merged**